packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -32,8 +34,8 @@ import 'constants.dart';
 ///
 /// class _MyTabbedPageState extends State<MyTabbedPage> with SingleTickerProviderStateMixin {
 ///   final List<Tab> myTabs = <Tab>[
-///     Tab(text: 'LEFT'),
-///     Tab(text: 'RIGHT'),
+///     Tab(text: 'left'),
```

Comment:
The convention per the Material spec is call caps for tab titles

Reply:
I reverted it to all caps